tests/pytorch_tests/model_tests/test_feature_models_runner.py

```diff
@@ -618,6 +617,13 @@ def test_gptq(self):
                               weights_quant_method=QuantizationMethod.UNIFORM,
                               params_learning=False).run_test()  # TODO: When params learning is True, the uniform quantizer gets a min value  > max value
 
+        # gradual activation quantization
+        GPTQAccuracyTest(self, gradual_activation_quantization=True).run_test()
```

Comment:
A general question:
this is basically not really a unit test, it tests the MCT workflow with gradual quantization.
How would you design this test in the new package? (hierarchywise, workflow wise, what it should test etc.)

If it is not part of this PR then I want us to plan how it should look like and implement such tests in a separate PR, because this is one of the main issues with changing our testing methodology.

Reply:
I suggest to open a designated task for this